spec/system/admin/access_code_groups_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Access codes admin menu", type: :system do
+  let(:organization) { create(:organization) }
+  let(:user) { create :user, :admin, :confirmed, organization: organization }
+
+  before do
+    switch_to_host(organization.host)
+    login_as user, scope: :user
+  end
+
+  it "adds a new access code group" do
+    visit decidim_admin_anonymous_codes.code_groups_path
+
+    click_on "New access code group"
+
+    fill_in_i18n(
+      :code_group_title,
+      "#code_group-title-tabs",
+      en: "New Group",
+      es: "NuevoGrupo",
+      ca: "Nou group"
+    )
+    fill_in "Expires At", with: (Time.zone.today + 1.day).strftime("%d/%m/%Y %H:%M")
+    check "Active"
+    fill_in "Re-use max", with: 10
+
+    click_on "create"
+
+    expect(page).to have_content("Access code group successfully created")
+    expect(page).to have_content("New access code group")
+
```

Comment:
```suggestion
last_group = Decidim::AnoynymousCodes::Group.last
expect(last_group.title["en]).to eq("New Group")
```
You can add here a check to see that the last group create has the proper introduced data

Reply:
Done